index.html

```diff
@@ -0,0 +1,88 @@
+</head>
+
+<body>
+    <div class="page">
+        <div class="container">
+            <header>
+                <ul>
+                    <li><a class="myLogo font-style" href="">Kenbon</a> </li>
+                    <nav class="font-style">
+                        <li><a class="toggler-icon"><img class="toggler-img" src="images/humburger.png"
+                                    alt="humberger-icon"></a>
+                        </li>
+                    </nav>
+
+                </ul>
+
+
+            </header>
+            <div class="main">
+                <section class="home-page">
+                    <section class="intro-section">
+                        <h1 class="main_heading font-style">I'm Kenbon <br> Glad to see you!</h1>
+                        <p class="description font-style">
+                            I’m a software developer! I can help you build a product , feature or
+                            website Look through some of my work and experience! If you like what
+                            you see and have a project you need coded, don’t hestiate to contact
+                            me.
+                        </p>
+                    </section>
+
+                    <section>
+                        <div class="connect-section">
+                            <h4 class="connect font-style">LET'S CONNECT</h4>
+                            <ul class="socials">
+                                <li>
+                                    <a class="social_medias twitter" href="https://twitter.com/KNurgi">
+                                        <img src="images/Vector (1).svg" alt="twitter logo">
+                                    </a>
+                                </li>
+    
+                                <li>
+                                    <a class="social_medias" href="https://www.linkedin.com/in/kenbon-teshome/">
+                                        <svg xmlns="http://www.w3.org/2000/svg" width="23" height="23" fill="currentColor"
+                                            class="bi bi-linkedin icon" viewBox="0 0 16 16">
+                                            <path
+                                                d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z" />
+                                        </svg>
+                                    </a>
+                                </li>
```

Comment:
- [ Required ] Please consider changing the `LinkedIn` Icon to match the background-color on the Figma template, In the  Figma template all the `icons` have the same background-color, whereas on your design the `LinkedIn` icon has a different color. Please see the screenshot below for more details on this.

## Figma template
![Screenshot from 2023-01-12 15-05-22](https://user-images.githubusercontent.com/16392046/212062186-2a657fc9-4d8a-4a54-81f4-af7e4ec0dd98.png)

## Your Implementation
![Screenshot from 2023-01-12 14-59-58](https://user-images.githubusercontent.com/16392046/212062259-4c1571f7-234b-4710-8f5e-1e7be693ecdd.png)